Assets/Scripts/ShadowPuzzle/PilarCheck.cs

```diff
@@ -0,0 +1,36 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEditor.Experimental.GraphView;
+using UnityEngine;
+using UnityEngine.Events;
+using System;
+
+public sealed class Pillars: MonoBehaviour
```

Comment:
```suggestion
public sealed class Pillar: MonoBehaviour
```
Verander ook de file dat het de class matched.